src/content/ja/updates/2019/02/rendering-on-the-web.md

```diff
@@ -165,37 +144,37 @@ SSRリハイドレーションを使用している実際のWebサイトから
-これは、クライアントサイドでページ内の優先度が低い部分をアップグレードした場合でも、メインスレッドがブロックされるのを防ぐことができるので、ページを対話可能とにするのに必要なJavaScript量の削減を助けることができます。
+プログレッシブ リハイドレーションもまた注目する価値があり、React が[探ってきた](https://github.com/facebook/react/pull/14717)ものもあります。  
+このアプローチでは、アプリケーションすべてを一度に初期化する現行の一般的なアプローチではなく、サーバーレンダリング アプリケーションの各部品が時間とともに“起動”されます。  
+これは、クライアントサイドでページ内の優先度が低い部分をアップグレードした場合でも、メインスレッドがブロックされるのを防ぐことができるので、ページを対話可能とにするのに必要なJavaScript量の削減を助けることができます。  
 また、サーバーレンダリングされたDOMツリーが破壊され、直ちに再構築された場合に、最も一般的なSSRリハイドレーションの落とし穴を回避するのを助けます - ほとんどの場合、初期の同期を行うクライアントサイド レンダリングにはまだ準備のできていないデータが必要であり、おそらくPromisの解決を待っている状態です。
 
 ### 部分的なリハイドレーション {: #partial-rehydration }
 
-部分的なリハイドレーションの実装は難しいということが裏付けられています。
-このアプローチは、段階的なリハイドレーションのアイディアの拡張で、段階的にリハイドレーションされるべき個々の部分（コンポーネント／ビュー／ツリー）が分析され、ほとんど対話可能でないものか、またはリアクティブではないものが識別されます。
-これらほとんどの静的な部分に対して、該当するJavaScriptのコードは、その時に不活性な参照と装飾になる機能に変換され、クライアント側のフットプリントは、ほぼ0へ縮小されます。
+部分的なリハイドレーションの実装は難しいということが裏付けられています。  
+このアプローチは、段階的なリハイドレーションのアイディアの拡張で、段階的にリハイドレーションされるべき個々の部分（コンポーネント／ビュー／ツリー）が分析され、ほとんど対話可能でないものか、またはリアクティブではないものが識別されます。  
+これらほとんどの静的な部分に対して、該当するJavaScriptのコードは、その時に不活性な参照と装飾になる機能に変換され、クライアント側のフットプリントは、ほぼ0へ縮小されます。  
 部分的なハイドレーションのアプローチは、自身の問題と妥協が伴います。それはキャッシングのための興味深い挑戦を示し、クライアントサイドのナビゲーションは、アプリケーションの不活性部分のためにサーバーレンダリングされたHTMLが、全ページのロードなしで使用可能になると想定できないことを意味します。
 
 ### トライソモルフィック レンダリング  {: #trisomorphic }
 
-もしも[サービスワーカー](/web/fundamentals/primers/service-workers/)をあなたのために選択するなら、“トライソモルフィック” レンダリングも面白いかもしれません。
+もしも[サービスワーカー](/web/fundamentals/primers/service-workers/)をあなたのために選択するなら、“トライソモルフィック” レンダリングも面白いかもしれません。  
 これは initial/non-JS ナビゲーションのためにストリーミング サーバー レンダリングを使うことができる時のテクニックで、それら（initial/non-JS）がインストールされた後、あなたのサービスワーカーはナビゲーションのためのHTMLレンダリングに取りかかります。
-これはキャッシュコンポーネントと最新のテンプレートを保持し、同一セッション内の新規ビューをレンダリングするためのSPAスタイルナビゲーションを有効にします。
+これはキャッシュコンポーネントと最新のテンプレートを保持し、同一セッション内の新規ビューをレンダリングするためのSPAスタイルナビゲーションを有効にします。  
 このアプローチは、同じテンプレートを生成したり、サーバー、クライアントページ、サーバーワーカーの間でルーティングコードを共有できる場合に適しています。
 
 <img src="../../images/2019/02/rendering-on-the-web/trisomorphic.png" alt="Diagram of Trisomorphic rendering, showing a browser and service worker
 communicating with the server">
 
 ## SEOについての考察 {: #seo }
 
-チームは Web上でレンダリングするための戦略を選択する場合、しばしば SEOの影響を考慮します。サーバーレンダリングはクローラが解釈しやすい“完全な見え方”のエクスペリエンスを提供するためによく選ばれます。
-クローラは [JavaScript を理解できます](https://web.dev/discoverable/how-search-works)が、それらはしばしば、どのようにレンダリングするかを認識している価値がある という[制約があります](/search/docs/guides/rendering)。
+チームは Web上でレンダリングするための戦略を選択する場合、しばしば SEOの影響を考慮します。サーバーレンダリングはクローラが解釈しやすい“完全な見え方”のエクスペリエンスを提供するためによく選ばれます。  
+クローラ[は JavaScript を理解でき](https://web.dev/discoverable/how-search-works)ますが、それらはしばしば、どのようにレンダリングするかを認識している価値がある と[いう制約があり](/search/docs/guides/rendering)ます。
 クライアントサイドレンダリングは機能しますが、大抵は追加テストや地道な調査が欠かせません。
-ごく最近では、クライアントサイドのJavaScriptによってアーキテクチャが大きく左右される場合、[ダイナミックレンダリング](/search/docs/guides/dynamic-rendering)もまた考慮する価値のある選択肢になりました。
+ごく最近では、クライアントサイドのJavaScriptによってアーキテクチャが大きく左右される場[合、ダイナミックレンダリ](/search/docs/guides/dynamic-rendering)ングもまた考慮する価値のある選択肢になりました。
```

Comment:
Odd link insertion.

Reply:
@svasilenkov if this is a result of Google Translation, let's ignore. Do you know?

Reply:
@agektmr Can't see anything odd with 3 links in this paragraph in the raw file from this commit:

https://raw.githubusercontent.com/google/WebFundamentals/31b9f81f8acc9ad2fcdf7aa243443c57c696e1af/src/content/ja/updates/2019/02/rendering-on-the-web.md

Could please you give more details on what do you think is odd here?